service/src/main/java/bio/terra/workspace/db/exception/ApplicationInUseException.java

```diff
@@ -0,0 +1,10 @@
+package bio.terra.workspace.db.exception;
+
+import bio.terra.common.exception.NotFoundException;
+
+public class ApplicationInUseException extends NotFoundException {
```

Comment:
404 seems off here, I think this should be something like 400

Reply:
I intended it to be BadRequest, but it was bad cut'n'paste.